docs/getting-started/installation-linux.md

```diff
@@ -150,6 +157,11 @@ $ npm install
 $ npm run build-assets
 ```
 
+> **Warning**
+>
+> Since version 2.9, it may be not working anymore because [Dashboard](https://github.com/mirumee/saleor-dashboard) and [Storefront](https://github.com/mirumee/saleor-storefront) are available through separate repositories : 
```

Comment:
Would maybe sound better like so?
```suggestion
> Since version 2.9, it may not be working anymore as [Dashboard](https://github.com/mirumee/saleor-dashboard) and [Storefront](https://github.com/mirumee/saleor-storefront) are available through separate repositories : 
```